docs/enriching-your-data/available-enrichments/custom-javascript-enrichment/examples/index.md

```diff
@@ -33,7 +33,43 @@ This example is not copy-paste-able. You will need to define your own schema.
 
 ## Filtering out bots
 
-You may want to prevent certain bot-generated events from entering your data warehouse. Here is an example that uses a [`throw` statement](/docs/enriching-your-data/available-enrichments/custom-javascript-enrichment/writing/index.md#discarding-the-event) to achieve that:
+You can use the [IAB enrichment](/docs/enriching-your-data/available-enrichments/iab-enrichment/index.md) to identify bots. However, it does not remove the bot-generated events from your data — it only adds relevant information to the events (in the `derived_contexts` field), e.g.:
+
+```json
+{
+    "schema": "iglu:com.iab.snowplow/spiders_and_robots/jsonschema/1-0-0",
+    "data": {
+        "spiderOrRobot": false,
+        "category": "BROWSER",
+        "reason": "PASSED_ALL",
+        "primaryImpact": "NONE"
+    }
+}
+```
+
+If you wish to discard the bot events (sending them to [failed events](/docs/managing-data-quality/failed-events/understanding-failed-events/index.md)), you can use the JavaScript enrichment:
+
+```js
```

Comment:
Nit: `javascript` is more common in the existing docs than `js` as a language tag, I think. Not sure if consistency there is useful or not.

Reply:
Would need to change it everywhere in these pages... I’d say not important :)